### ISSUE TITLE
ember-cli-babel 5.x deprecation

### ISSUE THREAD

**Issue by scottkidder** [poteto/ember-crumbly#135]
:tada:
```
warning ember-crumbly > ember-cli-babel@5.2.5: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6.0
warning ember-crumbly > ember-getowner-polyfill > ember-cli-babel@5.2.5: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6.0
```

**Comment by spacerobotTR** [comment]
I am facing the same issue also right now.

**Comment by zhangzhenhuajack** [comment]
has some problem

**Comment by onceatime** [comment]
X1

**Comment by scottkidder** [comment]
Fixed in #138 but there is one test still not passing if anyone can take a look!

**Comment by scottkidder** [comment]
Closed by #138
